modules/nw-egressnetworkpolicy-object.adoc

```diff
@@ -77,14 +77,16 @@ egress:
   to: <2>
     cidrSelector: <cidr> <3>
     dnsName: <dns_name> <4>
-  ports: <5>
+    nodeSelector: <label_name>: <label_value> <5>
+  ports: <6>
       ...
 ----
 <1> The type of rule. The value must be either `Allow` or `Deny`.
 <2> A stanza describing an egress traffic match rule that specifies the `cidrSelector` field or the `dnsName` field. You cannot use both fields in the same rule.
 <3> An IP address range in CIDR format.
 <4> A DNS domain name.
-<5> Optional: A stanza describing a collection of network ports and protocols for the rule.
+<5> A label that specifies which nodes to apply egress traffic rules to. Set the value to `true` to apply the rule.
```

Comment:
It's not really a value of "true"... a label is a key/value pair that can be anything. The nodeSelector allows for one or more node labels to be specified and selected upon. Like:

```
apiVersion: v1
kind: Pod

....

spec:
  nodeSelector: 
    region: east
    type: user-node
```

See: https://docs.openshift.com/container-platform/4.8/nodes/scheduling/nodes-scheduler-node-selectors.html